src/Cli/PhpsuApplication.php

```diff
@@ -3,23 +3,33 @@
 
 namespace PHPSu\Cli;
 
+use PHPSu\Helper\ApplicationHelper;
 use PHPSu\Config\ConfigurationLoader;
 use PHPSu\Controller;
 use Symfony\Component\Console\Application;
 
 final class PhpsuApplication
 {
-    public static function command(): void
-    {
-        self::createApplication()->run();
-    }
-
     public static function createApplication(): Application
     {
-        $application = new Application('phpsu', '1.0.0-alpha3');
+        $application = new Application('phpsu', (new ApplicationHelper())->getCurrentPHPSUVersion());
         $configurationLoader = new ConfigurationLoader();
-        $application->add(new SyncCliCommand($configurationLoader, new Controller()));
-        $application->add(new SshCliCommand($configurationLoader, new Controller()));
+        foreach (self::getAllCommandsInDirectory(__DIR__) as $class) {
+            $fqClass = __NAMESPACE__ . '\\' . $class;
+            $application->add(new $fqClass($configurationLoader, new Controller()));
```

Comment:
this magic is really unnecessary.

Reply:
Automatically registering commands in a namespace is not unnecessary imo